examples/preInit/prodigytech.groovy

```diff
@@ -0,0 +1,23 @@
+
+// Auto generated groovyscript example file
```

Comment:
i dont think this file is autogenerated anymore, comment should probably get removed